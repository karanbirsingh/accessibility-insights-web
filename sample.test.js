describe('sample', () => {
    it('test', () => {
        expect(3).toMatchSnapshot();
    });
});
