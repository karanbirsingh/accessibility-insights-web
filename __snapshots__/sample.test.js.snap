// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sample test 1`] = `3`;
